### PR TITLE
Simple express app to print Hello World on local host

### PR DESCRIPTION
**Pull request by isha-gautam**

### CODE REVIEW

File: feature-express-server/hello-world.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+// JavaScript source code
```
Comment (rgpt) [inline feature-express-server/hello-world.js]: Rename this file to "app.js" - This will be our entry point to the web app.

**Comment by rgpt** [comment]
We do not add node_modules over git as think of a scenario where we have a lot of node modules might be 1-1.5 GB size node modules .
so remove the node_modules folder and add it to .gitignore (this file states what all files/folder to not include while commiting or git status,etc git ignores them)

Doing `npm install` in the project downloads the project dependencies

**Comment by isha-gautam** [comment]
> Add the project's name as this branch will be merged to master so this package.json would be the main package.json

So, in the package.json file, I should make "name" as AnthroLink instead of featyre-express-server?

**Comment by rgpt** [comment]
@isha-gautam that is correct. This is like small features in separate branch will be developed and merged to master. So it won't disrupt the code in master even if we do something wrong. 

This type of format we will use in branches . 

mostly people keep master branch and a development branch and branches from development branch. so as soon as the code is complete in the feature branches it will be merged to development branch. And when development branch gets stable after testing and all it will merged to master and released. 
As we do not have complication like that we will keep master and child branches from master as of now. 

I just wanted to let you know how branches are handled generally.

**Pull request merged**